src/main/kotlin/domain/CardDeck.kt

```diff
@@ -0,0 +1,19 @@
+package domain
+
+class CardDeck {
+    private val cards: MutableList<Card> = mutableListOf()
+
+    init {
+        for (suit in Suit.entries) {
+            for (value in CardValue.entries) {
+                cards.add(Card(suit, value))
+            }
+        }
+        cards.shuffle()
+    }
+
+    fun drawCard(): Card {
+        check(cards.isNotEmpty()) { "카드가 더이상 없습니다." }
+        return cards.removeAt(cards.size - 1)
```

Comment:
요렇게 바꿔볼 수 있을 것 같아요~~
```suggestion
        return cards.removeLast()
```